mrchecker-framework-modules/mrchecker-database-module/src/test/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,25 @@
+<persistence xmlns="http://java.sun.com/xml/ns/persistence"
+             xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+             xsi:schemaLocation="http://java.sun.com/xml/ns/persistence http://java.sun.com/xml/ns/persistence/persistence_2_0.xsd"
+             version="2.0">
+
+    <persistence-unit name="mySqlExample" transaction-type="RESOURCE_LOCAL">
+        <description>
+            Definition of connection to MySql engine database.
+        </description>
+
+        <provider>org.hibernate.jpa.HibernatePersistenceProvider</provider>
+
+        <properties>
+            <property name="hibernate.dialect" value="org.hibernate.dialect.MySQLDialect"/>
```

Comment:
It would great to have one common settings file <settings.properties> -> . 
There for please use PropertiesFileSettings.java to inject variables into this xml file

Reply:
@przwojtkow What about Lukasz comment? Was it discussed as well?

Reply:
Discussed, created new task for that.